scripts/rebase-fork.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+if [[ -z $1 ]]; then
+    echo "Please specify the pull request number you want to rebase to a local branch."
+    echo "Usage: ./scripts/rebase-fork.sh <pr number>"
+    echo ""
+    echo "Example: ./scripts/rebase-fork.sh 557"
+    exit 1
+fi
+
+PR_NUMBER=$1
+
+declare -a req_tools=("gh" "git" "jq")
+for tool in "${req_tools[@]}"; do
+  if ! command -v "$tool" > /dev/null; then
+    echo "It looks like '${tool}' is not installed; please install it and run this script again."
+    exit 1
+  fi
+done
+
+# Check if the PR specified is a valid number
+re='^[0-9]+$'
+if ! [[ $PR_NUMBER =~ $re ]] ; then
+   echo "The PR you specify must be a valid integer number." >&2; exit 1
+fi
+
+# Check if the specified PR exists
+gh pr view $PR_NUMBER &> /dev/null
```

Comment:
I ran this command locally and saw an error:

```
(rebase-script) % gh pr view 557                                                                   ~/dev/go-tfe
GraphQL: Your token has not been granted the required scopes to execute this query. The 'login' field requires one of the following scopes: ['read:org'], but your token has only been granted the: ['repo'] scopes. Please modify your token's scopes at: https://github.com/settings/tokens., Your token has not been granted the required scopes to execute this query. The 'name' field requires one of the following scopes: ['read:org', 'read:discussion'], but your token has only been granted the: ['repo'] scopes. Please modify your token's scopes at: https://github.com/settings/tokens., Your token has not been granted the required scopes to execute this query. The 'slug' field requires one of the following scopes: ['read:org', 'read:discussion'], but your token has only been granted the: ['repo'] scopes. Please modify your token's scopes at: https://github.com/settings/tokens
```

I think it would be helpful to capture this error and let the user know they need to fix their token or login with gh.